includes/Infrastructure/HasRequirements.php

```diff
@@ -41,6 +41,8 @@ interface HasRequirements {
 	 * @since 1.11.0
 	 *
 	 * @return string[] List of required services.
+	 *
+	 * @phpstan-return class-string<Service>[] List of required services.
```

Comment:
Looks like I got this one wrong.

`get_requirements()` returns a list of service names, not class strings.

```suggestion
```